src/nnet3/nnet-diagnostics.h

```diff
@@ -49,14 +54,17 @@ struct NnetComputeProbOptions {
   // constructor of NnetComputeProb that takes a pointer to the nnet, and the
   // stats will be stored there.
   bool store_component_stats;
+  
+  bool compute_per_dim_accuracy;
```

Comment:
it would probably be more accurate to call this 'print_per_dim_accuracy', since I think you compute it in either case.